Shared/ViewModels/LibraryViewModel/NextUpLibraryViewModel.swift

```diff
@@ -7,11 +7,17 @@
 //
 
 import Combine
+import Defaults
 import Foundation
 import JellyfinAPI
 
 final class NextUpLibraryViewModel: PagingLibraryViewModel<BaseItemDto> {
 
+    @Default(.Customization.Home.maxNextUp)
```

Comment:
Don't use the property wrappers in non-view contexts. While they technically can still work, use the subscript instead at the usage sites.

Documentation:
[sindresorhus/Defaults#swiftui-support](https://github.com/sindresorhus/Defaults?tab=readme-ov-file#swiftui-support)